clients/ui/bff/internal/integrations/httpclient/http.go

```diff
@@ -111,7 +111,7 @@ func (c *HTTPClient) GET(url string) ([]byte, error) {
 	return body, nil
 }
 
-func (c *HTTPClient) POST(url string, body io.Reader) ([]byte, error) {
+func (c *HTTPClient) POST(url string, body io.Reader, contentType ...string) ([]byte, error) {
```

Comment:
@ppadti when the content Type will be ...string?

Can we follow https://pkg.go.dev/net/http#Client.Post signature?

Reply:
(we can go either with 1 or 0)